actions/admin/routes.go

```diff
@@ -39,6 +39,6 @@ func RegisterRoutes(router *apirouter.Router, appConfig *config.AppConfig, servi
 	router.HTTPRouter.POST("/"+config.APIVersion+"/admin/transactions/record", action.Request(router, require.Wrap(action.transactionRecord)))
 	router.HTTPRouter.POST("/"+config.APIVersion+"/admin/utxos/search", action.Request(router, require.Wrap(action.utxosSearch)))
 	router.HTTPRouter.POST("/"+config.APIVersion+"/admin/utxos/count", action.Request(router, require.Wrap(action.utxosCount)))
-	router.HTTPRouter.POST("/"+config.APIVersion+"/admin/xpubs/search", action.Request(router, require.Wrap(action.xpubsSearch)))
+	router.HTTPRouter.GET("/"+config.APIVersion+"/admin/xpubs/search", action.Request(router, require.Wrap(action.xpubsSearch)))
```

Comment:
Unfortunately currently there isn't HTTP method for "search",
and often POST is better, because it accepts body where you can put much longer "queries" then in url.
Although there is no limitation for URL length in specification, I saw that some network components could reject URL that is to long.
Therefore I'm voting for using POST in each search request and GET if this is get by id (or other single identifier). And if client is not working, it should be changed in a client, not here.